viam-cartographer/src/carto_facade_go/carto_facade.go

```diff
@@ -0,0 +1,278 @@
+// Package cartofacade provides an api to call into c code
```

Comment:
Lets change the directory structure a bit:

1. lets have all the code your created under a package called `cartofacade` 
2. this file should be moved (with the same content) to `cartofacade/cgo_api.go`